astroid/brain/brain_attrs.py

```diff
@@ -10,8 +10,18 @@
 from astroid.nodes.node_classes import AnnAssign, Assign, AssignName, Call, Unknown
 from astroid.nodes.scoped_nodes import ClassDef
 
-ATTRIB_NAMES = frozenset(("attr.ib", "attrib", "attr.attrib"))
-ATTRS_NAMES = frozenset(("attr.s", "attrs", "attr.attrs", "attr.attributes"))
+ATTRIB_NAMES = frozenset(("attr.ib", "attrib", "attr.attrib", "attr.field", "field"))
```

Comment:
I do not understand why there is "field" alone? 
By the way i wonder about the existence of `attrib` alone also.

Reply:
I guess this would help in cases where they are imported as `from attr import attrib` or `from attr import field`. Since this is only used to detect the attributes after an attrs-class is identified, I think the risk of a name collision should be negligible.

Reply:
Ok, thanks, that's what i thought too. I also think that there is no risk of name collision.